scripts/notarize.js

```diff
@@ -29,13 +29,25 @@ async function notarizeApp(context) {
   const appName = context.packager.appInfo.productFilename;
   const path = `${appOutDir}/${appName}.app`;
 
+  // if (!DEVELOPER_TEAM_ID) {
   await notarize({
     appBundleId: "com.ledger.live",
     appPath: path,
     ascProvider: "EpicDreamSAS",
     appleId: APPLEID,
     appleIdPassword: APPLEID_PASSWORD,
   });
+  // } else {
+  //   await notarize({
+  //     tool: "notarytool",
+  //     appBundleId: "com.ledger.live",
+  //     appPath: path,
+  //     ascProvider: "EpicDreamSAS",
+  //     appleId: APPLEID,
+  //     teamId: DEVELOPER_TEAM_ID,
+  //     appleIdPassword: APPLEID_PASSWORD,
+  //   });
+  // }
```

Comment:
comments normal ?

Reply:
Yes, I've left those comments because on the next XCode release, we will be able to use the new and supposedly faster notarization tool